var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -28,11 +28,10 @@
 class Flex(Package):
     """Flex is a tool for generating scanners."""
 
-    homepage = "http://flex.sourceforge.net/"
-    url = "http://download.sourceforge.net/flex/flex-2.5.39.tar.gz"
+    homepage = "https://github.com/westes/flex"
+    url = "https://github.com/westes/flex/releases/download/v2.6.2/flex-2.6.2.tar.gz"
 
-    version('2.6.0', '5724bcffed4ebe39e9b55a9be80859ec')
-    version('2.5.39', 'e133e9ead8ec0a58d81166b461244fde')
```

Comment:
@davydden, I've created #2385 to deal with this commit. I've also rebased this branch and removed the `flex` commits.